### PR TITLE
Added two courses R PT_BR

### PR DESCRIPTION
**Pull request by 42kkkkkaren**
Hello! I'm trying to help by adding courses that helped me!

## What does this PR do?
Add resource(s)

### Description
Added two courses that helped me in learning R

### Why is this valuable (or not)?
I learned a lot by watching this playlists

### How do we know it's really free?
I'ts from youtube!

### For book lists, is it a book? For course lists, is it a course? etc.
It's a course!

## Checklist:
- [ x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x ] Include author(s) and platform where appropriate.
- [ x] Put lists in alphabetical order, correct spacing.
- [x ] Add needed indications (PDF, access notes, under construction).
- [ x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-pt_BR.md [hunk 1]
```diff
@@ -30,6 +30,7 @@
 * [Node.js](#nodejs)
 * [PHP](#php)
 * [Python](#python)
+* [R](#R)
```
Comment (davorpa) [inline courses/free-courses-pt_BR.md:33]: - URL fragments goes in lowercase

```suggestion
* [R](#r)
```

File: courses/free-courses-pt_BR.md [hunk 2]
```diff
@@ -295,6 +296,10 @@
 * [Python para Iniciantes](https://www.udemy.com/python-para-iniciantes/) - Tiago Miguel (Udemy)
 * [Selenium com Python](https://www.youtube.com/playlist?list=PLOQgLBuj2-3LqnMYKZZgzeC7CKCPF375B) - Eduardo Mendes (YouTube)
 
+### R
+
+* [Curso de R com R Studio](https://www.youtube.com/watch?v=2mS-SzVWPZQ&list=PLzWDDw1w8cTS4i_B49WOWtjngjcMqTruG) - Escola de Inteligência Artificial
+* [Curso de R para iniciantes](https://www.youtube.com/watch?v=tfPsmDzS74c&list=PLyqOvdQmGdTQ5dE6hSD7ZGBu8bud70wYf) - Didática Tech
```
Comment (davorpa) [inline courses/free-courses-pt_BR.md:302]: - lint: leave 2 blank lines between listings and next heading

```suggestion

### R

* [Curso de R com R Studio](https://www.youtube.com/watch?v=2mS-SzVWPZQ&list=PLzWDDw1w8cTS4i_B49WOWtjngjcMqTruG) - Escola de Inteligência Artificial
* [Curso de R para iniciantes](https://www.youtube.com/watch?v=tfPsmDzS74c&list=PLyqOvdQmGdTQ5dE6hSD7ZGBu8bud70wYf) - Didática Tech

```

**Review by davorpa — CHANGES_REQUESTED** [review]

File: courses/free-courses-pt_BR.md [hunk 3]
```diff
@@ -296,6 +297,12 @@
 * [Selenium com Python](https://www.youtube.com/playlist?list=PLOQgLBuj2-3LqnMYKZZgzeC7CKCPF375B) - Eduardo Mendes (YouTube)
 
 
+### R
+
+* [Curso de R com R Studio](https://www.youtube.com/watch?v=2mS-SzVWPZQ&list=PLzWDDw1w8cTS4i_B49WOWtjngjcMqTruG) - Escola de Inteligência Artificial
```
Comment (davorpa) [inline courses/free-courses-pt_BR.md:302]: We prefer YT playlist URLs

```suggestion
* [Curso de R com R Studio](https://www.youtube.com/playlist?list=PLzWDDw1w8cTS4i_B49WOWtjngjcMqTruG) - Escola de Inteligência Artificial
```

File: courses/free-courses-pt_BR.md [hunk 4]
```diff
@@ -296,6 +297,12 @@
...
+* [Curso de R para iniciantes](https://www.youtube.com/watch?v=tfPsmDzS74c&list=PLyqOvdQmGdTQ5dE6hSD7ZGBu8bud70wYf) - Didática Tech
```
Comment (davorpa) [inline courses/free-courses-pt_BR.md:303]: - Use titles as same as provided by original source
- We prefer YT playlist URLs

```suggestion
* [Curso R para Iniciantes](https://www.youtube.com/playlist?list=PLyqOvdQmGdTQ5dE6hSD7ZGBu8bud70wYf) - Didática Tech
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books! Sorry for dealy - we've had over a thousand PRs!

**Pull request merged**